homework1_aboutme.html

```diff
@@ -0,0 +1,39 @@
+<!--homework1: about me page-->
+
+<html>
+	<head>
+		<title>About me</title>
+		<link rel="stylesheet" type="text/css" href="homework1_aboutme_styles.css">
+	</head>
+		
+		<body>
+
+
+			<div><hr color="#000000" size="4" width="1175px" align= "center"></div>
+			
+			<nav>
+				<i>About Me</i> | <a href="homework1_portfolio.html">Portfolio</a>
+			</nav>
+			
+			<h1>Joell Shepperson</h1>
+
+			<div><hr color="#000000" size="1" width="1025px" align= "center"></div>
+
+			<img src="images/jopic.jpg" alt="Joell Bio" height="250" width="250" align="right">
+
+			<h3>Lover of Innovation. Seeker of Collaboration.</h3>
+				
+			<p>Hi, I'm Joell! A passionate UX-er and developer-in-training. Currently residing in Washington, DC with a strong focus on bringing intuitive experiences and elegant design solutions and to the your fingertips. When I'm not tinkering with wireframes or HTML, you can find me outside hiking, camping and traveling.</p>
+			
+
+			<div><hr color="#000000" size="1" width="1025px" align= "center"></div>
+			
+			<div class="linkbar"><a href="https://www.facebook.com/joell.shepperson">Facebook</a> | <a href="http://linkedin.com/in/joellshepperson">LinkedIn</a> | <a href="http://instagram.com/jkshepperson">Instagram</a> | <a href="http://joell.carbonmade.com">UX Work</a></div>
```

Comment:
Excellent use of of the ```<div>``` tag with a class.  My only comment here is next time, put each nested element (the ```<a>``` tags on their own line - this will help a ton with the readability of your HTML file.

Also, just something to think about - these links could also be nested inside a ```<nav>``` tag, since they are somewhat of a "navigation" to your social media pages.  In practice, this really could go either way - it's usually a judgement call in my experience.